test/ui/POM/HomePage.ts

```diff
@@ -0,0 +1,32 @@
+import { WebDriver, By, WebElement } from 'selenium-webdriver';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
```

Comment:
It's not recommened to load the dotenv package as part of the code files. You can instead include it in the package.json commands for the UI tests as follows:

 ```
"test-ui:login": "mocha --require dotenv/config --require ts-node/register --grep '@login' test/ui/selenium/*.ts",
    "test-ui:roles": "mocha --require dotenv/config --require ts-node/register --grep '@roles' test/ui/selenium/*.ts",
```

You can then remove the dotenv in this file and the other POM/Selenium test files